src/Value/HealthReport.php

```diff
@@ -77,7 +77,9 @@ public function getStatusCode()
 
     public function jsonSerialize()
     {
-        $report['status'] = $this->status;
+        $report = [
+            'status' => $this->status
+        ];
```

Comment:
Was this an MD fail?

Reply:
Yes, `$report` wasn't initialized when `$report['status']` was set.